sonar-project.properties

```diff
@@ -69,7 +69,12 @@ sonar.coverage.exclusions=\
   src/utils/gCloudStorage.ts, \
   src/components/ArchivedComponents/**, \
   src/pages/pupils/beta/ab-demos/**, \
-  src/node-lib/posthog/**
+  src/node-lib/posthog/**, \
+  src/app/(registration)/**, \
+  src/app/layout.tsx, \
+  src/styles/oakThemeApp.ts, \
+  src/app/providers.tsx
```

Comment:
You can mark individual files/lines etc. to be ignored by lcov with istanbul comments https://github.com/istanbuljs/nyc?tab=readme-ov-file#parsing-hints-ignoring-lines.

Makes it a little easier to maintain if those files are deleted later.

Reply:
odd, i've used it elsewhere and it worked 😖 🤷🏻‍♂️ . Not a big deal.

Reply:
yeah I thought so, I wonder if perhaps it doesn't register now as the `ignore file` declaration isn't new code so isn't analysed